Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs

```diff
@@ -82,14 +85,19 @@ void OnGUI()
 		else
```

Comment:
A little nitpicky and not necessary, but I like trimming tge string, and then doing a string.isNullOrEmpty check,  just in case the string somwhow ends up null. (Sadly, isNullOrWhitspace no longer exists :( )